Module-6-Assignment/AccountContactsTest.cls

```diff
@@ -0,0 +1,59 @@
+/**
+ * Test class to test the functionality of AccountContacts Class 
+ */
+@isTest
+public class AccountContactsTest 
+{
+	@isTest
+    private static void testWithOneContactId()
+    {
+        Account acc = new Account(Name='Test Account', ContactIds__c = null);
+        insert acc;
+        Contact con = new Contact(LastName='Test contact', AccountId=acc.id);
+        insert con;
+        Test.startTest();
+        String resultantContactIds = AccountContacts.setContacts(con.id);
+        Test.stopTest();
+        System.assertEquals(con.id + ',', resultantContactIds);
```

Comment:
Extra comma should not be there at the end

Reply:
Done